cx_Freeze/macdist.py

```diff
@@ -171,9 +176,12 @@ def setRelativeReferencePaths(self):
                 if (name not in files and not path.startswith('/usr') and not
                         path.startswith('/System')):
                     print(referencedFile)
-                    self.copy_file(referencedFile,
-                                   os.path.join(self.binDir, name))
-                    files.append(name)
+                    try:
+                        self.copy_file(referencedFile, os.path.join(self.binDir, name))
+                    except DistutilsFileError as e:
+                        print(f"issue copying {referencedFile} to {os.path.join(self.binDir, name)} error {e} skipping")
```

Comment:
f-strings are new in Python 3.6. Can you replace this with code that works in Python 2.7, 3.5 and 3.6?

Reply:
sure thing... didn't think about backward compatibility for F strings